index.html

```diff
@@ -1,29 +1,152 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, intial-scale=1.0">
+        <Title>House of Shelby</Title>
+        <link href="style.css"rel="stylesheet" type="text/css">
+        <link href="reset.css"rel="stylesheet" type="text/css">   
+    </head>
+    <body>
+        <style>
+            body{
+                background-color:#100D0B;
+            }
+            
+        </style>
+        
+        <!--Navigation options for the website-->
+        <div class="Heading">
+            <p><a href="#">menu</a></p>
+            <p><a href="#">process</a></p>
+            <p><a href="#">barrels</a></p>
+            <!--Logo that will be used for the website-->
+            <div class="logo">
+                <img src="./images/shelby.png" alt="Logo Picture is here.">
+            </div>
+            
+            <p><a href="#">visit us</a></p>
+            <p><a href="#">our story</a></p>
+            <p><a href="#">family recipe</a></p>
+        </div>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+        <!--Landing Page introduction (First thing the users are gonna see)-->
+        <div class="layout1">
+            <!--Bar Image for the first layout-->
+            <div class="landImage">
+                <img src="./images/landingPageBar.png" alt="A Picture of a Bar.">
+                <!--First layer of Text-->
+                <h5 class="barText1">Embrace the <br>
+                    taste of the <br>
+                    Shelby's</h5>
+                <!--Second Layer  of Text-->
+                <h5 class="barText2">Embrace the <br>
+                    taste of the <br>
+                    Shelby's</h5>
+                <!--Button under the text/Call to action-->
+                <button class="rectangleBox"> Have a sip</button>
```

Comment:
Heading structure is an issue here. 

You need to follow logical heading order h1 ... h2 ... h3 .. h4 ... h5